core/src/test/java/hudson/FilePathTest.java

```diff
@@ -736,4 +737,38 @@ public void testEOFbrokenFlush() throws IOException, InterruptedException {
         // and now fail when flush is bad!
         tmpDirPath.child("../" + archive.getName()).untar(outDir, TarCompression.NONE);
     }
+
+    @Test public void deleteRecursiveOnUnix() throws Exception {
+        assumeFalse(Functions.isWindows());
```

Comment:
please add a string to the assume so that the reason is reporting in the results.